godump_test.go

```diff
@@ -891,6 +869,27 @@ func TestDumpWithCustomWriter(t *testing.T) {
 	}
 }
 
+func TestDumpWithCustomSkipStackFrames(t *testing.T) {
+	// caller stack frames are
+	//	1	godump.go           github.com/goforj/godump.findFirstNonInternalFrame			skip by initialCallerSkip
+	//	2	godump.go           github.com/goforj/godump.printDumpHeader					skip by initialCallerSkip
+	//	3	godump.go           github.com/goforj/godump.(*Dumper).DumpStr					skip by fail names contain godump.go
+	//	4	godump_test.go      github.com/goforj/godump.TestDumpWithCustomSkipStackFrames
+	//	5	testing.go          testing.tRunner
+	out := NewDumper().DumpStr("test")
+	assert.Contains(t, out, "godump_test.go")
+
+	out = NewDumper(WithSkipStackFrames(1)).DumpStr("test")
+	assert.NotContains(t, out, "godump_test.go")
+	assert.Contains(t, out, "testing.go")
```

Comment:
See

https://github.com/goforj/godump/pull/31#discussion_r2181952200

Here you are relying on the fact the internals are definitely and forever be in a file named testing.go in the std lib. You shouldn't.

Take a look at the solution I suggested in the message I mention